website/blog/2020-08-31-63updates.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Windows & macOS version updates, Hermes, and more"
+author: Kiki Saintonge, Harini Kannan, Steven Moyes
+authorURL: https://twitter.com/ReactNativeMSFT
+description: "React Native for Windows is now on a 0.63 stable release with React Native core, lighting up some fun new APIs"
+is_blog: true
+published_date: August 31, 2020
+---
+
+React Native for Windows is now on a [0.63]() stable release with React Native core, lighting up some fun new APIs.

```

Comment:
Where should this link point?

Reply:
Ah! good catch, this should just be pointing to the npm package.

Reply:
fixed!